test.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "oofers",
+	"description": "I made you bitch",
```

Comment:
Please refrain from using swears in a public repository.

Reply:
THAT FILE WAS NOT MEANT TO BE COMMITTED! You saw nothing.